src/clj/puppetlabs/services/request_handler/request_handler_core.clj

```diff
@@ -259,21 +260,35 @@
     (= (:type x)
        :puppetlabs.services.request-handler.request-handler-core/bad-request)))
 
+(defn jruby-timeout?
+  [x]
+  "Determine if the supplied slingshot message is for a JRuby borrow timeout."
+  (when (map? x)
+    (= (:type x)
+       ::jruby/jruby-timeout)))
```

Comment:
This doesn't seem like it should work, since `::jruby/jruby-timeout` here is equivalent to `::puppetlabs.services.request-handler.request-handler-core/jruby-timeout`, but since the `throw+` is in a different namespace,`